apis/addons/v1alpha1/addonoperators_types.go

```diff
@@ -0,0 +1,59 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	DefaultAddonOperatorName = "addon-operator"
+)
+
+// AddonOperatorSpec defines the desired state of Addon operator.
+type AddonOperatorSpec struct {
+}
+
+// AddonOperatorStatus defines the observed state of Addon
+type AddonOperatorStatus struct {
+	// The most recent generation observed by the controller.
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	// Conditions is a list of status conditions ths object is in.
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// Timestamp of the last reported status check
+	// +optional
+	LastHeartBeatTime metav1.Time `json:"lastHeartBeatTime"`
```

Comment:
Super NIT
```suggestion
	LastHeartbeatTime metav1.Time `json:"lastHeartbeatTime"`
```

Kube is writing heartbeat as a single word and looks like wiki agrees that it is a single word. https://en.wikipedia.org/wiki/Heartbeat
:heart: Compound words :)